infra/_modules/cms_function_app/storage_account.tf

```diff
@@ -0,0 +1,111 @@
+module "cms_storage_account" {
+  source = "github.com/pagopa/terraform-azurerm-v3//storage_account?ref=v8.26.3"
+
+  name                          = "${var.prefix}${var.env_short}${var.location_short}${var.domain}queuest01"
```

Comment:
**chore:** use `cms` instead of `queue` to indicate this storage account is useful for `cms` application (like `environment.app_name` in `azure_function_app` module)
```suggestion
  name                          = "${var.prefix}${var.env_short}${var.location_short}${var.domain}cmsst01"
```